src/css/templates/_nav.scss

```diff
@@ -194,6 +194,11 @@
     top: 15px;
     right: 15px;
   }
+
+  & > a.btn-emphasized.d-lg-block {
+    display: none !important;
+  }
```

Comment:
I didn't get this :man_shrugging:

Reply:
The `.d-lg-block` class does not leave choice.
Furthermore in this case, I though that is an acceptable solution as this is a specific element in a specific context so the impact here is really limited. (On desktop, when the menu is open, we hide the button).

But as a workaround, I could write myself the media query to show/hide button and link according to the viewport width.